pkg/cli/admin/prune/deployments/prune.go

```diff
@@ -60,11 +60,11 @@ func NewPruner(options PrunerOptions) Pruner {
 
 	filter := &andFilter{
 		filterPredicates: []FilterPredicate{
-			FilterDeploymentsPredicate,
```

Comment:
Are you sure you want to entirely remove that filter? Shouldn't that depend on the `Orphans` option?

Reply:
Yes, that's what made it work. That filter ignores the orphans option completely.

Reply:
What I'm saying is, that depending if `--orphans` flag was specified we should add the filter or not.